LifeHelper.Services/Areas/User/DTOs/UserRegisterDto.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace LifeHelper.Services.Areas.User.DTOs;
+
+public class UserRegisterDto
+{
+    public string Nickname { get; set; }
+    public string Email { get; set; }
+    public string Password { get; set; }
+    public string ConfirmPassword { get; set; }
+}
```

Comment:
remove confirm password field => it is unnecessary in the api case, so you can handle this only on the front-end side